src/scrabble.js

```diff
@@ -1,5 +1,76 @@
-function scrabble() {
+const createLetterValue = () => {
+  const letterScoreTable = {
+    ...Object.fromEntries(
+      ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'].map((key) => [key, 1])
+    ),
+    D: 2,
+    G: 2,
+    ...Object.fromEntries(['B', 'C', 'M', 'P'].map((key) => [key, 3])),
+    ...Object.fromEntries(['F', 'H', 'V', 'W', 'Y'].map((key) => [key, 4])),
+    K: 5,
+    J: 8,
+    X: 8,
+    Q: 10,
+    Z: 10
+  }
+  return letterScoreTable
+}
+
+const getLetterScore = (letter, multiplier, letterScoreTable) => {
+  if (letterScoreTable[letter.toUpperCase()] !== undefined)
+    return letterScoreTable[letter.toUpperCase()] * multiplier
+  else return 0
+}
+
+const manageWord = (word, letterScoreTable) => {
```

Comment:
quick question on how u can pass  the object name (letterScoreTable ) in function parameter ?

Reply:
the object is a type of variable in a way, so we can pass them as simple variables. Javascript will know that it's an object